source/includes/changestream-paradigm-warning.rst

```diff
@@ -1,4 +1,8 @@
 .. warning::
 
-   Attempting to use a ``ChangeStream`` as an ``Iterator`` that is in ``EventEmitter`` mode will cause an error,
-   and attempting to use a ``ChangeStream`` as an ``EventEmitter`` in ``Iterator`` mode will cause an error.
+   Using a ``ChangeStream`` in ``EventEmitter`` and ``Iterator`` mode
+   concurrently is not supported and causes an error. This is to prevent
```

Comment:
I think this is missing the subject and is therefore passive voice. To fix this, you can specify "the driver" as the subject: "... is not supported by the driver and..."